test-frame-test/src/test/java/io/skodjob/testframe/test/integration/ResourceManagerIT.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Skodjob authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.skodjob.testframe.test.integration;
+
+import io.fabric8.kubernetes.api.model.NamespaceBuilder;
+import io.skodjob.testframe.annotations.TestVisualSeparator;
+import io.skodjob.testframe.clients.KubeClusterException;
+import io.skodjob.testframe.resources.ResourceManager;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@io.skodjob.testframe.annotations.ResourceManager
+@TestVisualSeparator
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ResourceManagerIT {
```

Comment:
Should here be also checks in afterEach and afterAll that the particular namespaces are deleted?

Reply:
I can't use after each because it is called before callbacks so namespace exists until callback is called